core/src/main/java/org/mskcc/cbio/portal/scripts/ImportGenePanelProfileMap.java

```diff
@@ -39,24 +39,24 @@
 import java.io.*;
 import joptsimple.*;
 import java.util.*;
-import org.mskcc.cbio.portal.repository.GenePanelRepositoryLegacy;
 
 /**
  *
- * @author heinsz
+ * @author heinsz, sandertan
  */
 public class ImportGenePanelProfileMap extends ConsoleRunnable {
 
     private File genePanelProfileMapFile;
-    private static Properties properties;
     private String cancerStudyStableId;
+    private List<String> NON_GENE_PANEL_VALUES = Arrays.asList("NA", "WES", "WGS");
```

Comment:
see previous comment ... I think "NA" was requested to be removed as a valid option. My interpretation of the requests from @n1zea144 was that there be only a single value for "non gene panel":
private List<String> NON_GENE_PANEL_VALUES = Collections.singletonList("WES/WGS");
But maybe we should get clarification.